pkg/broadcaster/unbuffered.go

```diff
@@ -1,18 +1,18 @@
-package broadcastwriter
+package broadcaster
 
 import (
 	"io"
 	"sync"
 )
 
-// BroadcastWriter accumulate multiple io.WriteCloser by stream.
-type BroadcastWriter struct {
+// Unbuffered accumulate multiple io.WriteCloser by stream.
+type Unbuffered struct {
 	mu      sync.Mutex
 	writers []io.WriteCloser
 }
 
 // AddWriter adds new io.WriteCloser.
-func (w *BroadcastWriter) AddWriter(writer io.WriteCloser) {
+func (w *Unbuffered) AddWriter(writer io.WriteCloser) {
```

Comment:
I suggest renaming this to `Add` for consistency with the buffered version.

This version takes a `io.WriteCloser`, but the buffered version takes an `io.Writer`. I don't see any obvious way to harmonize this, though.